source/wp-content/themes/wporg-developer/inc/formatting.php

```diff
@@ -599,9 +599,9 @@ public static function fix_param_hash_formatting( $text ) {
 				if ( $name ) {
 					$new_text .= "<code>{$name}</code>";
 				}
-				$new_text .= "<span class='type'>{$type}</span><div class='desc'>{$description}</div>";
+				$new_text .= "<span class='type'>{$type}</span><div class='desc'>{$description}";
 				if ( ! $skip_closing_li ) {
-					$new_text .= '</li>';
+					$new_text .= '</div></li>';
```

Comment:
What happens when `$skip_closing_li` is true? It seems like there'd never be a `</div>`. Do we need to add that in one of the `$in_list` conditions?

I haven't found an example where anything looks broken, though.

Reply:
Yeah it could be broken if it ends up true, which _could_ be the case with bad data. We could get a nested object that has an opening `{` brace but the closing one never shows up. It will never close. I think it's a risk that has always existed and short of refactoring this whole section, I'm not certain there's an easy win. If `$skip_closing_li` doesn't return to false, I _think_ the `<li>` would never close as well.

Reply:
👍🏻